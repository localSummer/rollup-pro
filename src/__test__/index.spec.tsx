import * as React from 'react';
import { shallow } from 'enzyme';
import Counter from '../index';

it('test count', () => {
  const wrapper = shallow(<Counter/>);

  expect(wrapper.find('img')).toHaveLength(2);

  expect(wrapper).toMatchSnapshot();
})