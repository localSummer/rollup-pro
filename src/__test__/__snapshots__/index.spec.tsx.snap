// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test count 1`] = `
<div>
  <ShowCount
    count={0}
  />
  <img
    alt=""
    className="img"
  />
  <CountOperation
    onAdd={[Function]}
  />
  <img
    alt=""
    className="img"
  />
</div>
`;
